### ISSUE TITLE
oq-engine and oq-hazardlib must be installed via pip by deploy.sh

### ISSUE THREAD

**Issue by daniviga** [gem/oq-platform#615]
Since now we are not releasing packages for precise anymore we cannot relay on `apt` anymore.

We have to update the `deploy.sh` to use `pip` instead of `apt`:

``` bash
sudo pip install https://github.com/gem/oq-hazardlib/archive/engine-2.1.zip
sudo pip install https://github.com/gem/oq-engine/archive/engine-2.1.zip
```

**Comment by daniviga** [comment]
Probably we'll need a tag `stable` pointing to the latest stable release

**Comment by daniviga** [comment]
Closed by #627
